src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx

```diff
@@ -96,11 +93,16 @@ const EquipmentFilter = forwardRef(
         // fetching options in different criterias
         useEffect(() => {
             // Load voltage level IDs
-            fetchVoltageLevelsListInfos(studyUuid, currentNode.id)
+            fetchVoltageLevelsMapInfos(studyUuid, currentNode.id)
                 .then((voltageLevels) => {
                     const vlMap = new Map();
-                    voltageLevels.forEach((vl) => vlMap.set(vl.id, vl.name));
+                    const nvSet = new Set();
+                    voltageLevels.forEach((vl) => {
+                        vlMap.set(vl.id, vl.name);
+                        nvSet.add(vl.nominalV);
+                    });
                     setVoltageLevelsMap(vlMap);
+                    setNominalVoltages([...nvSet.values()].sort());
```

Comment:
sort is not effective for number, it seems sort for string

Reply:
Works on Chromium bases browsers 
![image](https://github.com/gridsuite/gridstudy-app/assets/75525996/58d79aba-c62f-48be-85e7-6c1b2ea392a5)

Reply:
My bad, I change it